scripts/vs/buildimage-vs-image/runtest.sh

```diff
@@ -62,7 +62,8 @@ snmp/test_snmp_queue.py \
 syslog/test_syslog.py \
 tacacs/test_rw_user.py \
 tacacs/test_ro_user.py \
-telemetry/test_telemetry.py"
+telemetry/test_telemetry.py \
+pc/test_po_cleanup.py"
```

Comment:
Can you keep the list in alphabetical order, please?

Reply:
@daall : Updated